analyze_caliper.py

```diff
@@ -17,82 +17,94 @@
 import pandas as pd
 from scipy import stats
 
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-f', '--cali_file', help="The cali file to be analyzed should go here")
+    parser.add_argument('parallel_type', choices=['raja', 'agency'], help="Parallelism framework being used.")
+    parser.add_argument('application_type', choices=['fft', 'mmult'], help="Application benchmark being plotted")
+    parser.add_argument('-o', '--output_file', help="Name of the output file")
+    parser.add_argument('-q', '--cali_query_loc', help="Path to cali query executable")
 
-parser = argparse.ArgumentParser()
-parser.add_argument('cali_file', help="The cali file to be analyzed should go here")
-args = parser.parse_args()
+    args = parser.parse_args()
+    if args.parallel_type == 'raja' and args.application_type == 'mmult':
+        print args.cali_query_loc
+        plot_raja_mmult(args.cali_file, args.output_file, args.cali_query_loc)
+    else:
```

Comment:
You just need to, ideally, add another function to handle your use case and then add a branch to this if/elif/else chain.